jenkins.sh

```diff
@@ -17,12 +17,12 @@ ls -1 | grep -v ^src | xargs -I{} mv {} src/$PROJ/
 go get github.com/tools/godep
```

Comment:
I wonder if this could be simplified by doing it in the make file since it's already got some of the steps

Reply:
we need to modify this jenkins script anyway as we're no longer using godep, but this was a weird way of doing this anyway - will change